data/lang/ui-core/en.json

```diff
@@ -1115,6 +1115,18 @@
     "description": "",
     "message": "Permission to launch denied: you have outstanding fines."
   },
+  "LAUNCH_PERMISSION_GRANTED": {
+    "description": "",
+    "message": "Launch permission granted."
+  },
+  "LAUNCH_PERMISSION_GRANTED_DEPART_QUICK": {
+    "description": "",
+    "message": "Permission granted. Please do not clutter the landing area and depart quickly."
```

Comment:
Maybe "please do not loiter ..." is more apt, than "clutter"?

Reply:
Actually, looking at the code, this is for when the station is under heavy load, which is a nice touch, but maybe it should be crammed in there as well, like "We're close to maximum traffic (docking?) capacity, please do not clutter/loiter ...."

Actually, perhaps "landing area" should be "docking (area?)", which I think might work better with both planet side and orbital stations?

Reply:
I had the same "docking"/"landing" dilemma here, and space stations have pads instead of docking ports, so I went with "landing".